packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -445,12 +445,12 @@ class SelectableRegionState extends State<SelectableRegion> with TextSelectionDe
   }
 
   void _startNewMouseSelectionGesture(DragDownDetails details) {
-    widget.focusNode.requestFocus();
     hideToolbar();
     _clearSelection();
   }
 
   void _handleMouseDragStart(DragStartDetails details) {
+    widget.focusNode.requestFocus();
```

Comment:
This looks reasonable, but I wonder if instead we should not request focus when the global position of the tap is not within the rects of one of the `Selectables` being managed by the `SelectionContainerDelegate`. cc @chunhtai if you have any thoughts on this.

A regression test that test the previously broken behavior will also be needed before merging this.

Reply:
Checking rects doesn't sounds useful, because it is possible that the drag starts at a place not touching selectables, then drag across them.

That said, I think the reason we need to requesting focus is to expand the selections with arrow key (through the actions mechanism). We might be able to delay requesting focus until one of the selectables says "I'm selected". What do you think @chunhtai ?

Reply:
yes This sounds good to me.